types/Card.ts

```diff
@@ -1,19 +1,35 @@
 import Product from '~/types/Product'
 
 export enum CardTypes {
-  INFO,
-  PRODUCT
+  INFO = 'info',
+  PRODUCT = 'product',
+  AD = 'ad'
 }
-interface Targets {
-  category?: string
-  product?: string
+
+export type Card = ProductCard | AdCard | InfoCard
```

Comment:
Is there a good reason why we combine product cards and extra cards?

Reply:
Hm, maybe not 🤔 I'll try another thing

Reply:
Ok, I understand

Reply:
Done 👍

Reply:
Do you like it?

Reply:
Yes! Much cleaner code, thanks!

Reply:
It's great to have someone to work together with 🙊